app/server/post_results.go

```diff
@@ -560,3 +546,18 @@ func getCertPool(cert []byte) (*x509.CertPool, error) {
 	}
 	return pool, nil
 }
+
+func (t tlogEntry) rekord() (hashedrekord.Body, error) {
+	b, err := base64.StdEncoding.DecodeString(t.Body)
+	if err != nil {
+		return hashedrekord.Body{}, fmt.Errorf("decode rekord body: %w", err)
+	}
+	var body hashedrekord.Body
+	if err := json.Unmarshal(b, &body); err != nil {
+		return hashedrekord.Body{}, fmt.Errorf("unmarshal rekord body: %w", err)
+	}
+	if body.Kind != hashedrekord.Kind {
```

Comment:
Just curious, what is `Kind`? It's not clear from the schema.

Reply:
https://github.com/sigstore/rekor/tree/main/pkg/types
> Rekor supports pluggable types (aka different schemas) for entries stored in the transparency log.

So Kind is a property of the base transparency log entry, and is an enum that identifies the specific type.